portfolio/src/main/webapp/script.js

```diff
@@ -11,18 +11,60 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 /**
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
+ 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+ 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+ 
+function getBookInfo() {
+ 
+    const urlHead = "https://www.googleapis.com/books/v1/volumes?q=";
+ 
+    fetch('/book-data').then(response => response.json()).then((bookInfo) => {
+        for (i = 0; i < bookInfo.length; i++) {
+            var query = bookInfo[i].title.replace(/ /g, "+");
+            var url = urlHead + query;
+            searchBooks(bookInfo[i], url);
+        }
+    });
+}
+ 
+function searchBooks(book, url)
+{
+   
+   $.getJSON(url, function (json)
+   {
+        var info = json.items[0].volumeInfo;
+        var title = book.title;
+        var reviews = book.reviews;
+        var genre = book.genre; // List of genres from GoodReads
+        var categories = info.categories; // List of genres from Books API
+        var author = info.authors;
+        var language = info.language;
+        var description = info.description; 
+        var infoLink = info.infoLink;
+        var pageCount = info.pageCount;
+        var publishedDate = info.publishedDate;
+        var publisher = info.publisher;
+        var maturityRating = info.maturityRating;
+ 
+        var booksHTML= title + " by " + author + " has maturity rating = " + maturityRating + ". Reviews: " + reviews;
+        console.log(booksHTML);
```

Comment:
is this for debuggin console.log?

Reply:
I was using it to demo the code. Removed :)